Deploy/stacks/KingsLynn/DTVF/data.json

```diff
@@ -419,6 +419,5617 @@
+                                                            ]
+                                                        ],
+                                                        0.0
+                                                    ],
+                                                    "<div order='102'>This point represents a cluster of closely spaced Sludge Sites.<br/></br/><span style='color: #DF6C35;'>In this case, at least one location has a non-zero chance of <b>indirect</b> failure.</span><br/><br/>Click to see details on the underlying individual locations.</div>",
+                                                    [
+                                                        "case",
+                                                        [
+                                                            "!=",
+                                                            [
+                                                                "-",
+                                                                1.0,
+                                                                [
+                                                                    "to-number",
+                                                                    [
+                                                                        "get",
+                                                                        "input_lowest_state"
+                                                                    ]
+                                                                ]
+                                                            ],
+                                                            0.0
+                                                        ],
+                                                        "<div order='103'>This point represents a cluster of closely spaced Sludge Sites.<br/></br/><span style='color: #7F7F7F;'>In this case, at least one location has a non-zero chance of secondary failure.</span><br/><br/>Click to see details on the underlying individual locations.</div>",
+                                                        ""
+                                                    ]
+                                                ]
+                                            ],
+                                            [
+                                                "accumulated"
+                                            ]
+                                        ]
+                                    }
+                                },
+                                {
+                                    "id": "sewage-source",
+                                    "type": "geojson",
+                                    "data": "http://kg.cmclinnovations.com:3839/geoserver/3LhiMsWN/ows?service=WFS&version=1.0.0&request=GetFeature&typeName=3LhiMsWN%3ASewage&outputFormat=application%2Fjson",
```

Comment:
Nice catch @cmclscourtney, this is something I'd forgotten about. Unfortunately, there's no nice solution here, CReDo has a _very_ specific implementation that was not designed to work out-of-the-box with the generic TWA infrastructure.

Dirtiest solution is just to tell @markushofmeister the new scenario IDs whenever we redeploy. The only real alternative is for Markus to go through the long & winding steps of creating a new agent to mimick CReDo's CentralStackAgent, updating all of his vis configs to load remotely, and using the CReDo specific version of the DTVF (something I would not advise, none of it is documented).

If I'm honest, I'm a little suprised we're trying to use the CReDo data outside of CReDo already; It _sounds_ like a good idea, but we didn't design this square peg to fit in round holes. I would personally adivse that we don't use the CReDo data outside of CReDo until we had time to review it, and (where possible) make generic versions of the features within the TWA.

Reply:
To clarify: The main reason for using the data is to plot it alongside the dynamic flood risk and impact evaluation within the King's Lynn use case to provide a sneak peak what else can be done in the very same system. Something along the lines: TWA can dynamically assess the poential impact of floods and at the same time provide more strategic planning and insights.
So, we mainly want to create one-two figures for the pre-print as current scope

Reply:
@cmclscourtney @mdhillman In case that's acceptable for you and you do not see any other issues, please approve so that I can merge those changes (as other people start to explore with the King's Lynn visualisation)